### PR TITLE
[INJIMOB-1709] -  Kotlin implementation if VC-Verifier

### PR DESCRIPTION
**Pull request by sree96**

### CODE REVIEW

File: vc-verifier/kotlin/vcverifier/publish-artifact.gradle [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+                username = System.getenv("OSSRH_USER")
+                password = System.getenv("OSSRH_SECRET")
+            }
+        }
+    }
+    publications {
+        aar(MavenPublication) {
+            pom {
+                withXml {
+                    def dependenciesNode = asNode().appendNode('dependencies')
```
Comment (swatigoel) [inline vc-verifier/kotlin/vcverifier/publish-artifact.gradle:16]: Please refer pixelpass to add all mandatory nodes and signing for release build

File: vc-verifier/kotlin/vcverifier/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,68 @@
+plugins {
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.jetbrainsKotlinAndroid)
+    `maven-publish`
+}
```
Comment (swatigoel) [inline vc-verifier/kotlin/vcverifier/build.gradle.kts:5]: add signing plugin, refer pixelpass

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: vc-verifier/kotlin/vcverifier/build.gradle.kts (hunk 2 above)
Comment (sree96) [reply vc-verifier/kotlin/vcverifier/build.gradle.kts:6]: Updated

File: vc-verifier/kotlin/vcverifier/publish-artifact.gradle (hunk 1 above)
Comment (sree96) [reply vc-verifier/kotlin/vcverifier/publish-artifact.gradle]: Updated

**Review by swatigoel — APPROVED** [review]

**Pull request merged**